pkg/component/etcd/configmap/values.go

```diff
@@ -56,10 +55,10 @@ type Values struct {
 	AutoCompactionMode *druidv1alpha1.CompactionMode
 	//AutoCompactionRetention defines the auto-compaction-retention length for etcd as well as for embedded-Etcd of backup-restore sidecar.
 	AutoCompactionRetention *string
-	// ConfigMapName is the name of the configmap that holds the ETCD config
-	ConfigMapName string
 	// ConfigMapChecksum is the checksum of deployed configmap
 	ConfigMapChecksum string
-	// Labels is the labels from the etcd object spec
+	// Labels is the labels of deployed configmap
```

Comment:
```suggestion
	// Labels are the labels of deployed configmap
```
Nit. Please verify this in other components as well, some need this change
Similar for `Annotations` as well